extra/3-magic-8-ball.js

```diff
@@ -103,22 +104,21 @@ console.log(shakeBall());
 
 function checkAnswer(answer, answerSelection) {
   //Write your code in here
-  let rating = "";
-   // if the "veryPositive" key values in the possibleAnswers object include the string assigned to "answer" at the index specified by "answerSelection", then return "rating" value very positive
+   // if the "veryPositive" values in the possibleAnswers object include the string assigned to "answer" at the index specified by "answerSelection", then return "very positive"
   if (possibleAnswers.veryPositive.includes(answer, [answerSelection])) {
```

Comment:
well done Kara. you used includes method as exactly as described in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes.  Your codes are brilliant!
:clap::clap:

Reply:
Thank you for all your helpful suggestions :)